modules/ROOT/pages/introduction-monitoring-metrics.adoc

```diff
@@ -105,54 +109,63 @@ However, the MicroProfile Metrics feature doesn't provide metrics about Open Lib
 
 If you enable both the MicroProfile Metrics and Performance Monitoring features, then the MXBeans for monitoring and the `/metrics` REST endpoint are activated.
 In this case, metrics for Open Liberty server components are exposed through both interfaces.
-The MicroProfile Metrics feature version 2.3 and later automatically enables the Performance Monitoring feature.
+The MicroProfile Metrics feature versions 2.3 and later automatically enable the Performance Monitoring feature.
 
-==  Integration with Micrometer
+[#micrometer]
+== Integration with Micrometer
```

Comment:
the list of micrometer integrations changes over time.  would be better not to specifically say "18"  monitoring systems (I currently count 19 on the page we link to).